<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sellerUi</class>
 <widget class="QWidget" name="sellerUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupSelect">
     <property name="title">
      <string>Report</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelFrom">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateFrom">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>d/M/yy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelTo">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDateEdit" name="dateTo">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2010</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="buttonOk">
        <property name="text">
         <string>Ok</string>
        </property>
        <property name="icon">
         <iconset resource="../../../core/res.qrc">
          <normaloff>:/conferma24x24.png</normaloff>:/conferma24x24.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QTableView" name="tableView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="simpleTable" name="tableSeller"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>simpleTable</class>
   <extends>QTableView</extends>
   <header location="global">simpletable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../core/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>sellerUi</receiver>
   <slot>okPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okPressed()</slot>
 </slots>
</ui>
